scripts/common/kubernetes.sh

```diff
@@ -52,16 +52,21 @@ function kubernetes_load_ipvs_modules() {
 }
 
 function kubernetes_sysctl_config() {
-    case "$LSB_DIST" in
-        # TODO I've only seen these disabled on centos/rhel but should be safe for ubuntu
-        centos|rhel|amzn|ol)
-            echo "net.bridge.bridge-nf-call-ip6tables = 1" > /etc/sysctl.d/k8s.conf
-            echo "net.bridge.bridge-nf-call-iptables = 1" >> /etc/sysctl.d/k8s.conf
-            echo "net.ipv4.conf.all.forwarding = 1" >> /etc/sysctl.d/k8s.conf
+    if ! grep -q "net.bridge.bridge-nf-call-ip6tables = 1" /etc/sysctl.conf; then
```

Comment:
This grep will fail if there isn't exactly one space character before and after the equal sign.  Could be better to split on `=` and trim spaces and check if value is not `1`